android/src/fdroid/play/version.yaml

```diff
@@ -0,0 +1 @@
+version: 2021.08.07-14-FDroid+21080714
```

Comment:
Probably we don't need this file:

> If no file for the version code was specified, code and name can be extracted from the tag: |\+(\d+)||Android-([\d.]+)

https://f-droid.org/en/docs/Build_Metadata_Reference/#UpdateCheckData

Reply:
I am still not sure. We need to try :)

Reply:
It looks like we still need it: https://gitlab.com/fdroid/fdroidserver/-/blob/master/fdroidserver/checkupdates.py#L145
Without this file, it will check build.gradle.